packages/components/psammead-headings/src/__snapshots__/index.test.jsx.snap

```diff
@@ -3,23 +3,24 @@
 exports[`Headline component should render correctly 1`] = `
 .c0 {
   color: #3F3F42;
-  font-family: ReithSerifNewsMedium,Helvetica,Arial,sans-serif;
+  font-family: ReithSerif,Helvetica,Arial,sans-serif;
   margin: 0;
   padding: 2rem 0 1rem 0;
-  font-size: 1.75em;
+  font-weight: 500;
```

Comment:
Should this have a font-weight?

Reply:
Spoke to Drew IRL and this matches the font-weight we have set on the font meaning it would switch to use the bold source file. Nominating @dr3 to create an issue to put these values into a package to be re-usable meaning it's less likely someone will use "faux bold" by mistake EG: `font-weight: 450;`

Reply:
Yeah, as weve gone from Medium

Reply:
![](https://media1.giphy.com/media/lu5LYc87R1960/giphy.gif?cid=3640f6095c7f97f65344422e55cb773c)

Reply:
Donneee https://github.com/bbc/psammead/issues/327